db/schema.rb

```diff
@@ -60,6 +69,22 @@
     t.integer "user_id", null: false
   end
 
+  create_table "schema_migrations", primary_key: "version", id: :string, force: :cascade do |t|
```

Comment:
what is this line ?

Reply:
This line gets created because i had to create a new schema manually. My first 'public' schema got deleted